lib/std/special/build_runner.zig

```diff
@@ -57,7 +57,8 @@ pub fn main() !void {
         break :x &stdout_file_stream.stream;
     } else |err| err;
 
-    while (arg_it.next(allocator)) |err_or_arg| {
+    var arg_index: usize = 5;
```

Comment:
I'm otherwise happy to merge this, but I think here it would be better to use `arg_it.index` rather than hard-coding this value 5.